nextcord/types/invite.py

```diff
@@ -6,6 +6,7 @@
 
 from typing_extensions import NotRequired
 
+from ..flags import InviteFlags
```

Comment:
This isn't going to work because the type of this parameter in the raw payload is just a simple integer. These typed dicts are supposed to represent the raw payloads, after all.